### PR TITLE
[new release] voqc (0.2.1)

### PR DESCRIPTION
**Pull request by khieta**
A verified optimizer for quantum circuits (VOQC)

- Project page: <a href="https://github.com/inQWIRE/mlvoqc">https://github.com/inQWIRE/mlvoqc</a>
- Documentation: <a href="https://inQWIRE.github.io/mlvoqc">https://inQWIRE.github.io/mlvoqc</a>

##### CHANGES:

- Updated to the most recent version of SQIR & extracted using Coq v8.13.2.
- The only change in SQIR that impacts the behavior of mlvoqc is that Optimize1qGates now merges as many gates as possible instead of just two at a time.

### CODE REVIEW

File: packages/voqc/voqc.0.2.1/opam [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+authors: ["Kesha Hietala"]
+license: "MIT"
+homepage: "https://github.com/inQWIRE/mlvoqc"
+doc: "https://inQWIRE.github.io/mlvoqc"
+bug-reports: "https://github.com/inQWIRE/mlvoqc/issues"
+depends: [
+  "dune" {>= "2.7"}
+  "ocaml" {>= "4.08"}
+  "openQASM"
+  "zarith"
```
Comment (mseri) [inline packages/voqc/voqc.0.2.1/opam:15]:
```suggestion
  "zarith" {>= "1.5"}
```

Comment (kit-ty-kate) [reply packages/voqc/voqc.0.2.1/opam]: For posterity: this package uses `Q.to_float` which was added in 1.5

**Comment by kit-ty-kate** [comment]
Thanks! Could you return the fix to the opam file upstream so this can be taken into account for the future releases?

**Comment by khieta** [comment]
Thanks! I've fixed the version requirement upstream.

**Pull request merged**